frontend/src/app/modules/fields/display/field-types/project-status-display-field.module.ts

```diff
@@ -32,7 +32,7 @@ import { projectStatusCodeCssClass, projectStatusI18n } from "core-app/modules/f
 
 export class ProjectStatusDisplayField extends DisplayField {
   public render(element:HTMLElement, displayText:string):void {
-    const code = this.value;
+    const code = this.value && this.value.id;
```

Comment:
```suggestion
    const code = this.value?.id;
```

Just a nitpick: We have safe operators in typescript as well.

Reply:
I just couldn't remember. Thanks for pointing it out.